client/src/services/DonuService.ts

```diff
@@ -21,7 +21,10 @@ export const fetchDonuModels = async (): Promise<any[]> => {
   const response = await callDonu(request);
   if (response.status === Donu.ResponseStatus.success) {
     const models = response?.result as Donu.ModelDefinition[] ?? null;
-    return donuToModel(models);
+    const grometOnlyModels = models.filter(model => {
+      return [Donu.Type.GROMET_PNC, Donu.Type.GROMET_PRT].includes(model.type);
```

Comment:
We don't want to support GROMET_PRT only `PNC` and the future `FN` SO I'd remove `Donu.Type.GROMET_PRT` from the list as well.